compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -121,7 +121,7 @@ macro_rules! maybe_recover_from_interpolated_ty_qpath {
     };
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq)]
```

Comment:
Why? I would prefer if people just write `matches!` here.

Reply:
tbh, no particularly strong reason, my bias is just to prefer to implement traits that seem applicable unless I can articulate my reservations about doing so, and I couldn't figure out that here.